harness/determined/common/api/authentication.py

```diff
@@ -70,6 +71,9 @@ def _init_session(
             self.token_store.drop_user(session_user)
             token = None
```

Comment:
it seems like at this point in the code, the `session_user` we set before will not longer be valid.